tests/components/axis/test_camera.py

```diff
@@ -31,11 +32,13 @@ async def test_camera(hass):
 
     assert len(hass.states.async_entity_ids(CAMERA_DOMAIN)) == 1
 
-    cam = hass.states.get(f"camera.{NAME}")
-    assert cam.state == "idle"
+    entity_id = f"{CAMERA_DOMAIN}.{NAME}"
+
+    cam = hass.states.get(entity_id)
+    assert cam.state == STATE_IDLE
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)